### DISCUSSION TITLE
Linkerd init startup error

### DISCUSSION THREAD

**Discussion by rishiravikumar-tul-scm** [linkerd/linkerd2#9040] [Help]
We see the following errors on some pods during startup, while other pods running on other nodes run fine. We are unable to debug why this seems to happen only when it's on a particular node. We have identified that the node is healthy. how can we resolve this?

Any help would be greatly appreciated!

linkerd-init container
```
time="2022-07-29T04:01:21Z" level=info msg="iptables-save -t nat"
time="2022-07-29T04:01:21Z" level=info msg="iptables-save v1.8.7 (legacy): Cannot initialize: Permission denied\n\n"
time="2022-07-29T04:01:21Z" level=error msg="aborting firewall configuration"
```

**Comment by mateiidavid** [comment]
@RishiRavikumar to troubleshoot issues, we generally require a bit more information about your environment, otherwise we can't really weigh in on what might be going wrong.

What environment are you using? Is it managed by a provider, do you manage it yourself? What Kubernetes version, what Linkerd version are you running, and what values are you using for your configuration? Are you running the proxy as root or non-root?

Do you use different security policies for your nodes, or something similar? Admin capabilities are baked into the binaries so you shouldn't require root privileges. Nodes that **use the docker runtime** are an exception to this.
